src/utils/selectors.ts

```diff
@@ -243,3 +250,36 @@ export const noopSelector: SparkleSelector<undefined> = () => undefined;
 
 export const emptyArray = [];
 export const emptyArraySelector = <T>(): T[] => emptyArray;
+
+// AnimateMap
+export const animateMapStageOptionsSelector: SparkleSelector<AnimateMapStageOptions> = (
+  state
+) => state.animatemap.stageOptions;
+
+export const animateMapWorldBoundsSelector: SparkleSelector<AnimateMapWorldBounds> = (
+  state
+) => {
+  return {
+    width: state.animatemap.worldWidth,
+    height: state.animatemap.worldHeight,
+  };
+};
+
+export const animateMapZoomSelector: SparkleSelector<number> = (state) =>
+  state.animatemap.zoom;
+
+export const animateMapUsersSelector: SparkleSelector<
+  Map<string, ReplicatedUser>
+> = (state) => state.animatemap.users;
+
+export const animateMapVenuesSelector: SparkleSelector<
+  Map<string, ReplicatedVenue>
+> = (state) => state.animatemap.venues;
+
+export const animateMapUsersQTSelector: SparkleSelector<QuadTree | null> = (
+  state
+) => state.animatemap.usersQT;
+
+export const animateMapVenuesQTSelector: SparkleSelector<QuadTree | null> = (
+  state
+) => state.animatemap.venuesQT;
```

Comment:
As called out earlier in review, `undefined` rather than `null`:

```suggestion
export const animateMapUsersQTSelector: SparkleSelector<QuadTree | undefined> = (
  state
) => state.animatemap.usersQT;

export const animateMapVenuesQTSelector: SparkleSelector<QuadTree | undefined> = (
  state
) => state.animatemap.venuesQT;
```